src/2_mockFn.test.js

```diff
@@ -1,7 +1,22 @@
 // Your tests here...
 // You'll practice the basics of jest mocks
-import { guardPower } from './2_mockFn';
+import { guardPower } from "./2_mockFn";
 
-it('...', () => {
-  // ...
+describe("guardPower()", () => {
+  it("should return cb(power) if power is less than 500", () => {
+    const mockCb = jest.fn();
+    const number = 3;
+    const power = 27;
+    guardPower(mockCb, number);
+    expect(mockCb).toHaveBeenCalled();
```

Comment:
suggestion: personally I like to always assert how many times a fn was called to prevent for example memory leaks or mistakes in recursive/re-renders scenarios:

```suggestion
    expect(mockCb).toHaveBeenCalledTimes(1);
```